filters/cropbyfocalpoint_test.go

```diff
@@ -118,8 +118,12 @@ func TestCropByFocalPoint_CreateFilter(t *testing.T) {
 		Args: []interface{}{0.5, 0.5, 1.5},
 		Err:  false,
 	}, {
-		Msg:  "more than 3 args",
-		Args: []interface{}{0.5, 0.5, 1.5, 1.0},
+		Msg:  "4 args",
+		Args: []interface{}{0.5, 0.5, 1.5, 200.0},
+		Err:  false,
+	}, {
```

Comment:
I would add a test to validate that if an image is smaller, then the focal point will be moved to ensure the resulting image has the desired dimensions. Basically that the dimension supersedes the specified focal point for smaller images. Thoughts?